tutorials/simulators/OpenFOAM.md

```diff
@@ -33,26 +31,10 @@ physical properties being modeled.
 - `system`: Contains files that describe the simulation, including solvers, 
 numerical parameters, and output files. 
 
-It must contain at least 3 files: 
-`controlDict`: Run control parameters (start/end time, time step, and parameters 
-for data output)
-`fvSchemes`: Selection of discretization schemes used during the solution.
-`fvSolution`: Equation solvers, tolerances and other algorithm controls.
+In order to run your simulation you cam simply run your `Allrun` script and you
```

Comment:
```suggestion
In order to run your simulation you can simply run your `Allrun` script and you
```
Should we give a brief explanation of what is the `Allrun` bash script?

Reply:
I'm not sure. If we are assuming they are openfoam users they should know what it is. What do you think @sarmento ?